Description: Add a brief summary to the GDAL and OGR utilities pages.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=421653
Author: Bas Couwenberg <sebastic@xs4all.nl>
Last-Update: 2013-10-13
--- a/apps/gdal_utilities.dox
+++ b/apps/gdal_utilities.dox
@@ -4,6 +4,7 @@
 
 /*! 
 \page gdal_utilities GDAL Utilities
+\brief A collection of GDAL related programs.
 
 The following utility programs are distributed with GDAL. 
 
--- a/apps/ogr_utilities.dox
+++ b/apps/ogr_utilities.dox
@@ -1,4 +1,5 @@
 /*! \page ogr_utilities OGR Utility Programs
+\brief A collection of OGR related programs.
 
 The following utilities are distributed as part of the OGR Simple Features
 toolkit:
